### PR TITLE
Disable merkle proofs for v2 get_block

### PR DESCRIPTION
**Pull request by antiochp**
This PR defaults to `include_merkle_proof = false` in the v2 api `get_block` jsonrpc endpoint.
These Merkle proofs are not actively used by anyone and were originally included (I believe) as a "proof of concept".
They are expensive to generate for historical blocks (specifically against a full archival node).

Related #3483.

----

Tested via - 

```
curl -X POST \
     -H 'Content-Type: application/json' \
     -d '{"jsonrpc":"2.0","id":1,"method":"get_block","params":[952019, null, null]}' \
         http://localhost:3413/v2/foreign
```

----

This is equivalent behavior to the optional `?no_merkle_proof` url param in the original v1 api endpoint - 

```
curl "localhost:3413/v1/blocks/952022?no_merkle_proof"
```

### CODE REVIEW

File: api/src/foreign_rpc.rs [hunk 1]
```diff
@@ -130,116 +130,125 @@ pub trait ForeignRpc: Sync + Send {
...
-				453337210,
-				459401597,
-				496068509,
-				511300624
+
+	curl -X POST \
+		 -H 'Content-Type: application/json' \
+		 -d '{"jsonrpc":"2.0","id":1,"method":"get_block","params":[952019, null, null]}' \
+		 http://localhost:3413/v2/foreign
+
```
Comment (quentinlesceller) [inline api/src/foreign_rpc.rs:142]: What's that? The doctests (currently disabled) are just below.

File: api/src/foreign_rpc.rs [hunk 2]
```diff
@@ -130,116 +130,125 @@ pub trait ForeignRpc: Sync + Send {
...
+
+
+		```
+		# grin_api::doctest_helper_json_rpc_foreign_assert_response!(
+		# r#"
+		{
+			"jsonrpc": "2.0",
+			"method": "get_block",
+			"params": [374274, null, null],
```
Comment (quentinlesceller) [inline api/src/foreign_rpc.rs:152]: This is the value you want to change here I guess

**Review by quentinlesceller — CHANGES_REQUESTED** [review]

**Comment by quentinlesceller** [comment]
Other than that everything looks good to me.

File: api/src/foreign_rpc.rs (hunk 1 above)
Comment (antiochp) [reply api/src/foreign_rpc.rs]: lol 😭

**Review by quentinlesceller — APPROVED** [review]

**Pull request merged**